docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,37 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::SerializableResource.serialize(post, options)
```

Comment:
Ahhhh. That definitely makes more sense. Probably going to go with that unless anyone wants to include both ways of doing it.